usagov/usagov-2021
# USAGOV-1801-uswds-textformat-a-tag-role-attribute: Allowed html tag update for USWDS text format

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1801

## Description

Update allowed html tags in USWDS text format config.

## Type of Changes

- [ X ] New Feature
  - Add screenshot showing what it should look like
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. Navigate to any page with an `<a>` tag in the WYSWYG. 
2. Edit the page, and using the source editor in the WYSWYG, add the `role="button"` property to the `<a>` tag.
3. Save and inspect the element on the page.
4. The element has the `role="button"` property present, and the link functions as expected.

### Change Requirements


- [ X ] Requires New Config

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
5. Find the commit of this pull request.
6. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
